internal/common/helper_test.go

```diff
@@ -0,0 +1,36 @@
+package common
+
+import (
+	"testing"
+	"time"
+)
+
+func TestRandomizeDuration(t *testing.T) {
+
+	tests := []struct {
+		name     string
+		variance float64
+		duration time.Duration
+	}{
+		{
+			name:     "returns valid duration in range",
```

Comment:
yep this is fine perhaps forcing it to do a loop and  a set number of iterations testing each is within the range?

Reply:
:facepalm: yep, that is smart. Done